### DISCUSSION TITLE
Vs2022 official version 17.0.0 can't create Maui?

### DISCUSSION THREAD

**Discussion by bbhxwl** [dotnet/maui#3311] [General] [answered]
Vs2022 official version 17.0.0 can't create Maui?

Didn't you say that the official version of Maui was released today?

**Comment by auriou** [comment]
Should we stay on the preview version ?

Is there an update to version "17.1.0 preview 1.0"

**Reply by bbhxwl** [reply]
I don't want to use the preview version, but only the preview version has Maui

**Comment by maxkoshevoi** [comment]
MAUI is in preview until Q2 2022, so it's not included in stable release of the VS yet.
But you can still open and build MAUI project's there.

Just run `maui-check` to setup everything:
```
dotnet tool update -g  redth.net.maui.check
maui-check
```

You can create MAUI template app with following commands:
```
dotnet new --install Microsoft.Maui.Templates
dotnet new maui -n HelloMaui
```

After that, you can open `HelloMaui` project in stable VS2022, and everything should work.

**Reply by bbhxwl** [reply]
> MAUI is in preview until Q2 2022, so it's not included in stable release of the VS yet. But you can still open and build MAUI project's there.
> 
> Just run `maui-check` to setup everything:
> 
> ```
> dotnet tool update -g  redth.net.maui.check
> maui-check
> ```
> 
> You can create MAUI template app with following commands:
> 
> ```
> dotnet new --install Microsoft.Maui.Templates
> dotnet new maui -n HelloMaui
> ```
> 
> After that, you can open `HelloMaui` project in stable VS2022, and everything should work.

![image](https://user-images.githubusercontent.com/26525599/141036500-9695bbe8-d4e0-47f8-8874-d59986bd6831.png)

**Reply by maxkoshevoi** [reply]
What does the errors say? I think you have some unresolved issues in `maui-check`.

Could you please provide a screenshot of `maui-check`? It should look like this:
![image](https://user-images.githubusercontent.com/6609929/141085449-ce37cad5-fa23-4ee1-a0a8-ffb12d5b819d.png)

(there should be only one issue - `Missing Visual Studio >= 17.1.0-pre.1.0`, the rest should be green. If there are more issues fix them using `maui-check`)

**Comment by davidbritch** [comment, marked as answer]
You now need Visual Studio 17.1 (Preview) to create MAUI apps.

**Comment by workgroupengineering** [comment]
Microsoft Visual Studio Community 2022
Version 17.1.0 Preview 1.0
VisualStudio.17.Preview/17.1.0-pre.1.0+31903.286
Microsoft .NET Framework
Version 4.8.04084

Window 10 21H1 Intel(R) Core(TM) i7-7700K CPU

When i try to install maui-android is reported `Workload ID android isn't supported on this platform.` 

How can I solve?
